### PR TITLE
holo-keygen: drop

### PR DESCRIPTION
**Pull request by yegortimoshenko**
`holo-keygen` was used to initialize Holochain keystore before `hpos-state-keystore-derive` was implemented.

### CODE REVIEW

**Comment by yegortimoshenko** [comment]
If you want to work on this, please make a separate PR. With each orthogonal change to `2019-11-12/ci`, it becomes harder and harder to get it merged.

**Comment by pjkundert** [comment]
I've pulled the gist of the changes into this branch, so we can discuss them and/or merge them.

The re-addition of the Hydra tests for `envoy` await the merging of the Nix-ified new envoy code...

File: tests/holo-envoy/default.nix [hunk 1]
```diff
@@ -5,18 +5,18 @@ makeTest {
 
   machine = {
     imports = [ (import ../../profiles/holoportos/demo) ];
-    virtualisation.memorySize = 2048;
```
Comment (pjkundert) [inline tests/holo-envoy/default.nix]: This change brings the tests into line with what we discovered during `holoportos-shell demo` testing; sometimes, the keystore derivation uses up all the memory and fails, on a 2GB VM.  When we start testing envoy again, we'll need to do keystore derivation.

File: tests/holo-envoy/default.nix [hunk 2]
```diff
@@ -5,18 +5,18 @@ makeTest {
...
   testScript = ''
     startAll;
-    $machine->succeed("holo-keygen /var/lib/holochain-conductor/holo");
-    $machine->systemctl("restart holochain-conductor.service");
-    $machine->systemctl("start holo-envoy.service");
-    $machine->waitForUnit("holochain-conductor.service");
-    $machine->waitForUnit("holo-envoy.service");
-    $machine->waitForOpenPort("1111");
-    $machine->waitForOpenPort("48080");
+    # TODO: Revise these when the new envoy is integrated
```
Comment (yegortimoshenko) [inline tests/holo-envoy/default.nix]: Could you drop `tests/holo-envoy/default.nix` and document this [on our issue tracker](https://github.com/Holo-Host/holo-nixpkgs/issues)?

Comment (samrose) [reply tests/holo-envoy/default.nix]: @yegortimoshenko added to https://github.com/Holo-Host/holo-nixpkgs/issues/193

**Comment by yegortimoshenko** [comment]
31a3532519f3738196489a53ad654dc086a2fb0c doesn't conform to [`CONTRIBUTING.md`](https://github.com/Holo-Host/holo-nixpkgs/blob/develop/CONTRIBUTING.md) and contains an unaddresssed requested change. It shouldn't have been merged in. Things like this are disproportionately slowing us down long-term.

**Pull request merged**